shared/unattended/RHEL-8-devel.ks

```diff
@@ -0,0 +1,100 @@
+install
+KVM_TEST_MEDIUM
+GRAPHICAL_OR_TEXT
+poweroff
+lang en_US.UTF-8
+keyboard us
+network --onboot yes --device eth0 --bootproto dhcp
+rootpw --plaintext 123456
+firstboot --disable
+user --name=test --password=123456
+firewall --enabled --ssh
+selinux --enforcing
+timezone --utc America/New_York
+bootloader --location=mbr --append="console=tty0 console=ttyS0,115200"
+zerombr
+KVM_TEST_LOGGING
+clearpart --all --initlabel
+autopart
+xconfig --startxonboot
+# Additional repositories could be specified in 'kickstart_extra_repos' parameter
+KVM_TEST_REPOS
+
+%packages --ignoremissing
+@base
+@core
+@development
+@additional-devel
+@debugging
+@network-tools
+@gnome-desktop
+@fonts
+@smart-card
+python3-pillow
+python3-six
+python3-pyparsing
+net-tools
+NetworkManager
+dconf
+watchdog
+coreutils
+usbutils
+spice-gtk3
+docbook-utils
+sgml-common
+openjade
+virt-viewer
+pulseaudio-libs-devel
+mesa-libGL-devel
+pygtk2-devel
+libjpeg-turbo-devel
+spice-vdagent
+usbredir
+SDL
+totem
+dmidecode
+alsa-utils
+sg3_utils
+-gnome-initial-setup
+%end
+
+%post
+# Output to all consoles defined in /proc/consoles, use "major:minor" as
+# device names are unreliable on some platforms
+# https://bugzilla.redhat.com/show_bug.cgi?id=1351968
+function ECHO { for TTY in `cat /proc/consoles | awk '{print $NF}'`; do source "/sys/dev/char/$TTY/uevent" && echo "$*" > /dev/$DEVNAME; done }
+ECHO "OS install is completed"
+ECHO "remove rhgb quiet by grubby"
+grubby --remove-args="rhgb quiet" --update-kernel=$(grubby --default-kernel)
+if [ $(arch) == "ppc64le" ]; then
+    grubby --args="console=hvc0,38400" --update-kernel=$(grubby --default-kernel)
+fi
+ECHO "dhclient"
+dhclient
+ECHO "systemctl sshd enable"
```

Comment:
Although, it is only a print info, but better keep consistent with the true command line.
`systemctl enable sshd`